easybuild/main.py

```diff
@@ -182,6 +188,25 @@ def handle_github_options(options, ec_paths):
     return done
 
 
+def _use_color(colorize, stream=sys.stdout):
```

Comment:
Bcc:
Reply-To:
(Kenneth Hoste, Fri, Mar 25, 2016 at 03:46:34PM -0700:)
> > @@ -182,6 +188,25 @@ def handle_github_options(options, ec_paths):
> >      return done
> >
> >
> > +def _use_color(colorize, stream=sys.stdout):
>
> this should probably be moved into `tools/options.py`,

Done.

> and be called from the `postprocess` there to determine whether `options.color` should be defined as `True` or `False`, based on the provided (string) value?

Not done: `fancylogger.logToScreen()` expects a 3-choice string value so
I'm keeping it this way.